### DISCUSSION TITLE
[Question] Is it possible to get a gradient opacity effect in an image?

### DISCUSSION THREAD

**Discussion by KazysNoobiys** [SixLabors/ImageSharp#2071] [Ideas]
In a mobile app on Xamarin.Forms I need to use an image with a gradient opacity. For example, the top of the image would be completely non-transparent and gradually become transparent to the bottom.
In WPF, there is an OpacityMask that allows you to do this. Xamarin doesn't have it. Is it possible to use
ImageSharp to generate an image with the required transparency on the server?

**Comment by ynse01** [comment]
Sounds to me, you'll need functionality from ImageSharp.Drawing. 
Have a look at, for example, the `LinearGradiantBrush` class [there](https://github.com/SixLabors/ImageSharp.Drawing/blob/main/src/ImageSharp.Drawing/Processing/LinearGradientBrush.cs).

**Reply by JimBobSquarePants** [reply]
I would suggest simply looping through the image rows and lerping the alpha component.
